### ISSUE TITLE
Translation not loading correctly when connected to woo.com

### ISSUE THREAD

**Issue by peterfabian** [woocommerce/woocommerce#27307]
**Prerequisites (mark completed items with an [x]):**
- [x] I have have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

**Describe the bug**
Screencast from @dougaitken :
https://d.pr/v/ltRJh2

It seems like it's not always broken like this, but needs further investigation.

**Expected behavior**
Translations are loaded.

**Actual behavior**
Translations are not loaded.

p1597233979280700-slack-C0E1AV8T0

**Comment by juliaamosova** [comment]
I did a bit of testing on this issue and I couldn't reproduce the exact behavior related to the fact that translations are not loading correctly when the site is connected to woo.com account. 

But I was able to encounter another translation issue that could be related?

Tested on brand new JN site with:

- WooCommerce 4.3.2 (installed by searching the plugin via `Plugins/Add New`)
- WordPress - 5.5

I tested Catalan and Russian languages. With both languages, I got WooCommerce Core translated fully after updating translations on `Dashboard/Updates/Translation` but the WooCommerce Admin screens don't seem to be translating. 

Catalan:

WooCommerce Core:

![catalan](https://user-images.githubusercontent.com/19143190/90067092-9af60c00-dcbc-11ea-8f55-0636d8b1e50b.jpg)

WooCommerce/Home (provided via WooCommerce Admin package in Core):

![catalan_home](https://user-images.githubusercontent.com/19143190/90067114-a0ebed00-dcbc-11ea-8cf4-507163e73d59.jpg)

Russian:

WooCommerce Core:

![russian](https://user-images.githubusercontent.com/19143190/90067147-a9442800-dcbc-11ea-92a6-2ffbf6cfcc9b.jpg)

WooCommerce/Home (provided via WooCommerce Admin package in Core):

![russian_home](https://user-images.githubusercontent.com/19143190/90067168-af3a0900-dcbc-11ea-84d4-4531c64b6243.jpg)

I further installed Loco Translate plugin to see if the strings of the Home screen are a part of the WooCommerce plugin and they don't seem to be for either Catalan or Russian. I searched for several strings that don't get translated but the example below is with the `Finish setup` string:

![cat_tra](https://user-images.githubusercontent.com/19143190/90068001-f2e14280-dcbd-11ea-8b5c-5bbb55f60f18.jpg)

![ru_tra](https://user-images.githubusercontent.com/19143190/90068012-f70d6000-dcbd-11ea-88cd-87a3ca460be2.jpg)

I'll keep researching further. Wanted to document the steps I took so far.

**Comment by dougaitken** [comment]
@juliaamosova - this WC Admin translation issue was due to a bug that is now resolved. I chatted with @jeffstieler about this yesterday - p1597237351358400-slack-C9K5T0XSP

For this main issue I wasn't able to replicate again, I've been testing again on two other sites I have and no matter the sequence of events, I can't replicate this. 

I'm unsure if I was using a translation that was less than 100% and then when I disconnected I had it changed to a language that was 100% to the translation installed as expected. 

I'm really sorry for the extra effort here to @peterfabian and the team but until this can be replicated, let's close it off.
